pkg/cmd/admin/registry/registry.go

```diff
@@ -299,6 +303,9 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 	env := app.Environment{}
 	env.Add(secretEnv)
 
+	if cfg.EnforceQuota {
+		env[server.EnforceQuotaEnvVar] = "true"
```

Comment:
Don't take constant references across code boundaries.  Admin CLI code should never take a reference to the docker registry code.  This should just be a string.